lib/resources/majors.tsx

```diff
@@ -2572,18 +2720,30 @@ const bsCS_New: Major = {
         'Students are required to fulfill the university’s requirement of two writing intensive courses, ' +
         'each at least 3 credits. Students must receive at least a C- grade or better in these writing courses. ',
       criteria: 'Written Intensive[W]',
```

Comment:
writing intensives can overlap with any distribution I think, but this works

Reply:
I thought not including anything means by default everything works